release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -466,19 +466,115 @@ With this update, the default credentials request for AWS has been modified to a
 
+[id="ocp-4-13-storage-user-managed-encryption"]
+==== User-managed encryption is generally available
+The user-managed encryption feature allows you to provide keys during installation that encrypt {product-title} node root volumes, and enables all managed storage classes to use these keys to encrypt provisioned storage volumes. This allows you to encrypt storage volumes with your selected key, instead of the platform’s default account key.
+
+This features supports the following storage types:
+
+* Amazon Web Services (AWS) Elastic Block storage (EBS) (for more information, see ???)
+
+* Microsoft Azure Disk storage (for more information, see ???)
+
+* Google Cloud Platform (GCP) persistent disk (PD) storage (for more information, see ???)
+
+[id="ocp-4-13-storage-csi-vol-detach-non-graceful-shutdown"]
+==== Detach CSI volumes after non-graceful node shutdown (Technology Preview)
+Container Storage Interface (CSI) drivers can now automatically detach volumes when a node goes down non-gracefully. When a non-graceful node shutdown occurs, you can then manually add an out-of-service taint on the node to allow volumes to automatically detach from the node. This feature is supported with Technology Preview status.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vol-detach-non-graceful-shutdown.adoc[Detach CSI volumes after non-graceful node shutdown].
+
+[id="ocp-4-13-storage-vsphere-encryption-support"]
+==== VMware vSphere encryption support is generally available
+You can encrypt virtual machines (VMs) and persistent volumes (PVs) on {product-title} running on vSphere. 
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#vsphere-pv-encryption[vSphere persistent disks encryption].
+
+[id="ocp-4-13-storage-vpshere-top-support-multi-datacenters"]
+==== VMware vSphere CSI topology support for multiple datacenters is generally available
+{product-title} 4.12 introduced the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple compute clusters, thus helping to avoid a single point of failure. {product-title} 4.13 introduces support for deploying over multiple datacenters and to set up the topology using failure domains created during installation or post-installation.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-top-aware_persistent-storage-csi-vsphere[vSphere CSI topology].
+
+[id="ocp-4-13-storage-multiple-default-sc"]
+==== Creating more than one default storage class is generally available
```

Comment:
@lpettyjo shouldn't this section reference the doc changes you made here, instead of `Managing the default storage class`?
https://github.com/openshift/openshift-docs/pull/55822/files

Reply:
@dobsonj Yes and no. I was directing all of the SC stuff to that topic because it is actually a catch-all for all of the SC changes in 4.13. However, your point is valid in that it takes you to a top-level topic and this actual feature is somewhat buried in there, so I've changed it.